blocks.js

```diff
@@ -7553,13 +7554,41 @@ InputSlotMorph.prototype.dropDownMenu = function (enableKeyboard) {
         if (Object.prototype.hasOwnProperty.call(choices, key)) {
             if (key[0] === '~') {
                 menu.addLine();
-            // } else if (key.indexOf('§_def') === 0) {
```

Comment:
It's a long, sad story.  I meant to type C-J to start a new line properly indented, but instead hit M-J because my hand slipped, so all the spacing of this "paragraph" was lost and I had to recreate it.  It's better now.